lib/react.mli

```diff
@@ -0,0 +1,146 @@
+[@@@warning "-67"] (* There's an unused parameter on functor Fragment *)
```

Comment:
you could probably add it to the type itself 

```ocaml
type x = ... [@@warning "-67"]
```

Reply:
Totally, I don't understand the error since complains about a functor where I don't have one.

Thanks for sneaking on the Pars :P